### DISCUSSION TITLE
gesture swipe left (or right) to add song as next in queue

### DISCUSSION THREAD

**Discussion by ChickensioDev** [fast4x/RiMusic#402] [Ideas]
id love to see a gesture like in spotify where if they swipe left on a song from some playlist itll add it next to queue

also id love an option where u can turn off 50 songs getting into queue when u play some random song so that i can just search and swipe songs in order to make my desired queue, and this option can extend for playlists, wherein if u play a song from a playlist, only that song plays and u can scroll around the playlist and swipe left to add other songs in ur desired order

also QOL feature for the first one, if u swipe left on one song it adds it next to queue, but if u swipe left on a second song, it can go below the first song which got added to queue as it makes more sense than going above the first song but ya thats just qol and not that necessary, id love to see the gesture tho!

**Comment by fast4x** [comment]
Hi, add this request as feature request on issue section, please.
Thanks for request.

**Reply by ChickensioDev** [reply]
ok sure! ill just copy paste it ig

**Reply by fast4x** [reply]
Yes
